src/nat/front_ends/fastapi/utils.py

```
@@ -0,0 +1,57 @@
+    Raises ImportError if the class cannot be imported.
+    """
+    try:
+        class_name_parts = class_full_name.split(".")
+
+        module_name = ".".join(class_name_parts[:-1])
+        class_name = class_name_parts[-1]
+
+        module = importlib.import_module(module_name)
+
+        if not hasattr(module, class_name):
+            raise ValueError(f"Class '{class_full_name}' not found.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)